subsys/nrf_security/src/drivers/cracen/cracenpsa/src/sign.c

```diff
@@ -466,15 +466,17 @@ static psa_status_t cracen_signature_ecc_verify(bool is_message,
 		return PSA_ERROR_INVALID_SIGNATURE;
 	}
 
-	if (alg == PSA_ALG_ED25519PH) {
```

Comment:
Can we use the PSA_NEED symbols here?
PSA_NEED_CRACEN_ED25519PH
PSA_NEED_CRACEN_PURE_EDDSA_TWISTED_EDWARDS_255

would do the same thing as what you added. 

I see that we have some other is_enabled with the PSA_WANT in this file, ideally we want to convert all the driver configuration to the PSA_NEED symbols. But this is a change for another time, you can just update the ones relevant to you at the moment.

Reply:
I can yes, and for EDDSA it is not a problem, but for a lot of algorithms we don't have good configs for it. As an example for ecdsa we don't have a "PSA_NEED_CRACEN_ECDSA" we only have:`PSA_NEED_CRACEN_ECDSA_BRAINPOOL_P_R1_192`
or `PSA_NEED_CRACEN_ECDSA_BRAINPOOL_P_R1_256` and I wanted to avoid having a wall of `or` statements. I could go through and write configs for it.

Reply:
I am aware of this indeed. I tried to create group configs wherever I can but the list is for sure not complete. For brainpool actually there is on grouped config `PSA_NEED_CRACEN_ECDSA_BRAINPOOL_P_R1`.

In my opinion we should just create more of these as needed in order to have a sensible configuration in the driver. It is more clear to me to have only PSA_NEED to configure parts inside the driver compare to having multiple configurations.  But thats a generic discussion I guess.

Reply:
Makes sense. Added in a ecdsa and deterministic_ecdsa config and updated sign.c to only use `NEED` configs